### PR TITLE
Governance Polls 2024-10-07

### PR DESCRIPTION
**Pull request by blimpa**
@VoteWizard

### CODE REVIEW

**Comment by VoteWizard** [comment]
Great job, @blimpa!

File: governance/polls/SparkLend Ethereum - Onboard PT-sUSDe-26Dec2024 and PT-sUSDe-27Mar2025 to Morpho Spark DAI Vault - October 7, 2024.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+**This is a binary vote.**
+
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to onboarding the two additional Pendle sUSDe PT maturities PT-sUSDe-26Dec2024 and PT-sUSDe-27Mar2025 to the Morpho Spark DAI vault with the configurations that follow, as [the currently listed Pendle sUSDe PT](https://morpho.blockanalitica.com/ethereum/markets/8f46cd82c4c44a090c3d72bd7a84baf4e69ee50331d5deae514f86fe062b0748) is close to expiration:
```
Comment (MakerJanSky) [inline governance/polls/SparkLend Ethereum - Onboard PT-sUSDe-26Dec2024 and PT-sUSDe-27Mar2025 to Morpho Spark DAI Vault - October 7, 2024.md:30]:
```suggestion
The community can vote in this poll to express support or opposition to onboarding the two additional Pendle sUSDe PT maturities PT-sUSDe-26Dec2024 and PT-sUSDe-27Mar2025 to the Morpho Spark DAI vault with the configurations that follow:
```

Comment (MakerJanSky) [reply governance/polls/SparkLend Ethereum - Onboard PT-sUSDe-26Dec2024 and PT-sUSDe-27Mar2025 to Morpho Spark DAI Vault - October 7, 2024.md:30]: This additional context has the potential to bias voters. While not incorrect, we do not believe it is appropriate for the review section.

File: governance/polls/SparkLend Ethereum - Onboard PT-sUSDe-26Dec2024 and PT-sUSDe-27Mar2025 to Morpho Spark DAI Vault - October 7, 2024.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+### PT-sUSDe-27Mar2025
+
+- Address: **[0xE00bd3Df25fb187d6ABBB620b3dfd19839947b81](https://etherscan.io/token/0xE00bd3Df25fb187d6ABBB620b3dfd19839947b81)**
+- Pool LLTV: **91.5%**
+- Supply cap: **100 million DAI**
+- Price oracle yield to maturity: **20%**
+- Price oracle PT redemption value: **$1**
+
+Please review the discussion [thread](https://forum.sky.money/t/oct-3-2024-proposed-changes-to-spark-for-upcoming-spell/25293) to help inform your position before voting. An overview of Pendle PTs can be found in [this July 13, 2023 forum post](https://forum.sky.money/t/jul-12-2024-proposed-changes-to-spark-for-upcoming-spell/24635#pendle-pt-overview-1).
```
Comment (VoteWizard) [inline governance/polls/SparkLend Ethereum - Onboard PT-sUSDe-26Dec2024 and PT-sUSDe-27Mar2025 to Morpho Spark DAI Vault - October 7, 2024.md:48]:
```suggestion
Please review the discussion [thread](https://forum.sky.money/t/oct-3-2024-proposed-changes-to-spark-for-upcoming-spell/25293) to help inform your position before voting. An overview of Pendle PTs can be found in [this July 13, 2024 forum post](https://forum.sky.money/t/jul-12-2024-proposed-changes-to-spark-for-upcoming-spell/24635#pendle-pt-overview-1).
```

**Review by MakerJanSky — APPROVED** [review]
Stylistic suggestion only.

**Comment by MakerJanSky** [comment]
# Poll Review

## Admin I

- [X] Is the poll expected this week?
- [X] Do you understand the question the poll is asking of governance?

## Content

- [X] Does the filename have the .md extension?
- [X] Does the filename contain banned characters? <>:"/\?*|
- [X] Do the titles contain banned characters?  <>:"/\?*
- [X] Are the poll options too long?  (Less than 30 characters as a rough guide)
- [X] Do the options have distinct numbers in the preamble?
- [X] Are the start and end dates present and correct?
- [X] Does the vote type match desired vote type for the poll?
- [X] Is the default option correct in metadata?
- [X] Is the preamble ready for submission?
- [X] Is the vote descriptor present and does it match the vote type?
- [X] Are the 'Review', 'Outcomes', and 'Resources' sections present?
- [X] Will the Data/parameters present in the poll be accurate at the time the poll is live?
- [X] Are the dates correct within the text?
- [X] Are all the relevant links provided?
- [X] Are all the links working?
- [X] Are the outcomes clear and comprehensive?
- [X] Do any '$' parameter signifiers remain?
- [X] Does the poll summary accurately reflect the poll content?
- [X] Read the poll text in full and highlight any other potential issues.

## Admin II

- [X] Raise any issues with the poll author via review on GitHub.
- [X] Inform the reviewer when the poll passes review.

File: governance/polls/SparkLend Ethereum - Onboard PT-sUSDe-26Dec2024 and PT-sUSDe-27Mar2025 to Morpho Spark DAI Vault - October 7, 2024.md (hunk 1 above)
Comment (VoteWizard) [reply governance/polls/SparkLend Ethereum - Onboard PT-sUSDe-26Dec2024 and PT-sUSDe-27Mar2025 to Morpho Spark DAI Vault - October 7, 2024.md:30]: We agree, thanks JS!

**Pull request merged**